pkg/cmd/server/origin/master.go

```diff
@@ -447,15 +452,22 @@ func (c *MasterConfig) Run(protected []APIInstaller, unprotected []APIInstaller)
 	open.Handle("/", handler)
 
 	// install swagger
+	// Expose files in third_party/swagger-ui/ on <host>/swagger-ui
+	fileServer := http.FileServer(&assetfs.AssetFS{Asset: kui.Asset, AssetDir: kui.AssetDir, Prefix: "third_party/swagger-ui"})
```

Comment:
Your enthusiasm is appreciated.  That'll be gone soon. :)

----- Original Message -----
> > @@ -447,15 +452,22 @@ func (c *MasterConfig) Run(protected []APIInstaller,
> > unprotected []APIInstaller)
> >  	open.Handle("/", handler)
> >  
> >  	// install swagger
> > +	// Expose files in third_party/swagger-ui/ on <host>/swagger-ui
> > +	fileServer := http.FileServer(&assetfs.AssetFS{Asset: kui.Asset,
> > AssetDir: kui.AssetDir, Prefix: "third_party/swagger-ui"})
> 
> > Yeah, you have to do a bunch of other stuff too.
> 
> That'll teach me to stick my fingers where they don't belong. :)
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1892/files#r29044742